src/components/alerts.css

```diff
@@ -0,0 +1,61 @@
+.alert {
+  width: 100%;
+  background-color: var(--black-100);
+  color: var(--black-500);
+  padding: var(--sp-md) var(--sp-lg);
+  font-size: var(--tx-20);
+  font-weight: var(--tx-medium);
+  border-radius: var(--radius-sm);
+  box-shadow: var(--shadow-smooth);
+  margin: var(--sp-lg) 0;
+  display: flex;
```

Comment:
You can try to store these flex properties in a `helper/utilities` file and directly use the classnames

Reply:
actually, i don't want to write long classnames for just alert. like it will become a long one e.g.


> `<div class="alert alert-primary d-flex flex-center"></div>` 
i think this is not a good approach in my view. but i would love to know your approach.